modules/culling/src/algorithms/bounding-box-from-points.js

```diff
@@ -108,7 +108,8 @@ export function makeOrientedBoundingBoxFromPoints(positions, result = new Orient
     .add(v3);
 
   const scale = scratchVector3.set(u1 - l1, u2 - l2, u3 - l3).multiplyByScalar(0.5);
-  result.halfAxes.multiplyByScalar(scale);
+  const scaleMatrix = new Matrix3([scale[0], 0, 0, 0, scale[1], 0, 0, 0, scale[2]]);
```

Comment:
Just `vector.scale(scale)`?

https://github.com/uber-web/math.gl/blob/master/modules/core/src/classes/base/math-array.js#L174

Reply:
If I do `const scaleMatrix = new Matrix3().scale(scale);` It won't work because of the same problem which I described above.
Any matrix scaling by using vector doesn't work correctly. Therefore I create the scale matrix instead.